pkg/setting/setting.go

```diff
@@ -200,6 +200,8 @@ var (
 type Cfg struct {
 	Raw *ini.File
 
+	Env string
```

Comment:
Added Env to Cfg. Again not sure if it makes that much sense for this particular variable (ie whether it makes sense to hot reload it)